rebuild_all_base.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+declare -A cuda_versions_python=(
```

Comment:
Why `declare -A` here ?

Reply:
`declare -A` is used to declare an associative array in Bash. Associative arrays are collections of key/value pairs, where each key is unique and is used to access its corresponding value. They are useful for storing and retrieving elements where the order of elements is not important, or when you want to access elements via specific keys rather than by a numeric index.